controllers/administration.js

```diff
@@ -2778,17 +2778,24 @@ const getTeamMembersButton = (counter) => `
 const getTeamSchoolsButton = (counter) => `
   <div class="btn-show-schools" role="button">${counter}<i class="fa fa-building team-flags"></i></div>`;
 
-router.all('/teams', (req, res, next) => {
+router.all('/teams', async (req, res, next) => {
 	const path = '/administration/teams/';
 
 	const itemsPerPage = parseInt(req.query.limit, 10) || 25;
 	const filterQuery = {};
 	const currentPage = parseInt(req.query.p, 10) || 1;
 
+	const dataLength = await api(req)
+		.get('/teams/manage/admin')
+		.then((dataResponse) => dataResponse.total );
+
+	const exceedDataLimit = ((itemsPerPage * (currentPage - 1)) > dataLength) ? true : false;
```

Comment:
Do you need that ? true : false; ?
```suggestion
	const exceedDataLimit = ((itemsPerPage * (currentPage - 1)) > dataLength);
```
It should also work the same way.